ansible/role_flux.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Manage local FluxCD
```

Comment:
We should change the name to be consistent all over the task names with the market term for Flux v2 -> Flux2

Reply:
I will update that in the next commit.